include/grpc/grpc_security.h

```diff
@@ -530,6 +548,26 @@ GRPCAPI grpc_server_credentials* grpc_ssl_server_credentials_create(
     const char* pem_root_certs, grpc_ssl_pem_key_cert_pair* pem_key_cert_pairs,
     size_t num_key_cert_pairs, int force_client_auth, void* reserved);
 
+/** Sets the minimum TLS version that will be negotiated during the TLS
+   handshake. The caller MUST ensure that |creds| is an instance of
+   |grpc_ssl_server_credentials|.
+    - creds: an instance of |grpc_ssl_server_credentials|. If |creds| is
+      nullptr, then this is a no-op.
+    - min_tls_version: the minimum TLS version. */
+GRPCAPI void grpc_ssl_server_credentials_set_max_tls_version(
```

Comment:
It looks like we declare `grpc_ssl_server_credentials_set_max_tls_version` twice?